refactor(orc8r): remove old northbound listSubscribers handler

christinewang5 opened the pull request:
<!--
    feat(orc8r): remove old northbound listSubscribers handler
-->

## Summary
* Remove old northbound listSubscribers handler
* Updated tests to reflect change

## Test Plan
* ran tests 
![image](https://user-images.githubusercontent.com/25142344/136109862-285ed51c-ec36-4be3-a62a-bce63de719d4.png)

christinewang5 committed:
refactor(orc8r): remove old northbound listSubscribers handler

christinewang5 committed:
feat(orc8r): remove old northbound listSubscribers handler

hcgatewood commented on lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers_test.go:
```diff
@@ -2149,14 +1762,17 @@ func TestSubscriberPolicy(t *testing.T) {
 	tests.RunUnitTest(t, e, tc)
 
 	// Get all, updated subscriber matches the expected value
+	expected = subscriberModels.PaginatedSubscribers{TotalCount: 1, NextPageToken: "", Subscribers: map[string]*subscriberModels.Subscriber{
```
Nit: move the fields of the PaginatedSubscribers each to their own line

hcgatewood reviewed (commented):
Looks great, thanks for adding!

I think to finish it out let's go ahead and just use the original endpoint name (so use the new endpoint, but no "v2" in the name). So make the change in the Go code (and validate at the rest api as you have so far), then for the NMS:

1. Search for `v2\b` (or just `v2`) in `nms/packages/magmalte/`, and you'll want to replace the subscribersv2 names with subscribers
2. [Run NMS precommit tests](https://magma.github.io/magma/docs/next/nms/dev_testing#pre-commit-tests)
3. [Spin up your local NMS](https://magma.github.io/magma/docs/next/basics/quick_start_guide#using-the-nms-ui) and sanity check (create a subscriber then delete it) the [subscribers page](https://magma.github.io/magma/docs/next/nms/subscriber)

cc @andreilee

pshelar committed:
chore(agw): update magma package dependency (#9471)

Update OVS and libfluid deps.

Signed-off-by: Pravin B Shelar <pravin.ovn@gmail.com>

Co-authored-by: Pravin B Shelar <pravin.ovn@gmail.com>

pshelar committed:
fix(pipelined): ipv4 neighbour discovery handling. (#9444)

Following patch fixes neighbour discovery request from
SGi interface.

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

ulaskozat committed:
fix(mme): remove memory leaks on sctp socket creation (#9481)

* fix(mme): remove memory leaks on sctp socket creation

Signed-off-by: Ulas Kozat <kozat@fb.com>

ulaskozat committed:
fix(mme): remove memory leak on migrated esm ebr timer (#9475)

* fix(mme): remove memory leak on migrated esm ebr timer

Signed-off-by: Ulas Kozat <kozat@fb.com>

pruthvihebbani committed:
fix(mme): Fix for handling ULA failure (#9460)

* Fixed sending of duplicate attach reject and ue context rel

Signed-off-by: Pruthvi Hebbani <pruthvi.hebbani@radisys.com>

VinashakAnkitAman committed:
fix(agw): [Non_Sanity] Fixed enb reset test cases getting stuck (#9108)

Signed-off-by: VinashakAnkitAman <ankit.aman@radisys.com>

VinashakAnkitAman committed:
feat(agw): [Non_Sanity] Added missing test case to defs.mk and updated non-sanity (#9099)

Signed-off-by: VinashakAnkitAman <ankit.aman@radisys.com>

ulaskozat committed:
fix(mme): bug fix for proto_to_guti_table conversion (#9485)

* fix(mme): bug fix for proto_to_guti_table conversion

Signed-off-by: Ulas Kozat <kozat@fb.com>

ulaskozat committed:
fix(mme): eliminate memory leaked during config parsing (#9487)

Signed-off-by: Ulas Kozat <kozat@fb.com>

themarwhal committed:
feat: add system level bazelrcs to make building easier in various systems (#9441)

Signed-off-by: Marie Bremner <marwhal@fb.com>

themarwhal committed:
chore: insert space in yml to see if this is the issue for docker push job (#9489)

Signed-off-by: Marie Bremner <marwhal@fb.com>

christinewang5 committed:
replaced listSubscribers v2 handler

christinewang5 committed:
replaced listSubscribers v2 handler

quentinDERORY committed:
feat(ci): migrate lte integ test to github actions (#9492)

First version, will later upload test results and optimize it by not building in the job

Signed-off-by: quentinDERORY <15911421+quentinDERORY@users.noreply.github.com>

quentinDERORY committed:
fix(ci): specify macOS-10 (#9501)

few tweaks to make it work in GHA

Signed-off-by: quentinDERORY <15911421+quentinDERORY@users.noreply.github.com>

themarwhal committed:
chore: update devcontainer image to sha-6c9b920 (#9491)

Signed-off-by: GitHub <noreply@github.com>

hcgatewood commented on lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml:
```diff
@@ -40,50 +40,6 @@ info:
-        default:
-          $ref: './orc8r-swagger-common.yml#/responses/UnexpectedError'
-
   /lte/{network_id}/subscribers_v2:
```
Need to remove the `_v2` from this for the swagger ui

<img width="1792" alt="Screen Shot 2021-10-06 at 2 05 36 PM" src="https://user-images.githubusercontent.com/8029544/136283260-59845729-a490-44c3-9943-dd4b69e38fa4.png">

christinewang5 replied:
removed!

![image](https://user-images.githubusercontent.com/25142344/136291624-c9773840-af71-44cd-892e-6904ed452730.png)

hcgatewood reviewed (changes requested):
Okay! As discussed offline, I pulled down your PR and validated the NMS changes. Everything lgtm!

Let's make that 1 swagger.v1.yml change (you'll have to build.py --generate again), then I'm ready to stamp and ship 🙂 

<img width="1792" alt="Screen Shot 2021-10-06 at 2 05 21 PM" src="https://user-images.githubusercontent.com/8029544/136283153-214b4eed-d21a-42bf-8800-3b27bb281b99.png">

hcgatewood commented:
Side notes:

1. To pass the semantic pr check, prepend `refactor: ` to your PR title
2. You'll have to pull and rebase onto latest master to reconcile the merge conflict in lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers_test.go

christinewang5 committed:
removed v2 handlers from swagger UI

christinewang5 committed:
removed v2 handlers from swagger UI

christinewang5 committed:
merged to resolve conflicts

christinewang5 commented:
closing because there is a new pr with the correct rebasing: 
https://github.com/magma/magma/pull/9567
